src/digraph.rs

```diff
@@ -61,6 +61,7 @@ use super::{
 /// (although rarely used for edges) is indexed by an integer id. These ids
 /// are stable for the lifetime of the graph object and on node or edge
 /// deletions you can have holes in the list of indices for the graph.
+/// Node indices will be reused on additions after removal.
```

Comment:
Can we add the example from the release note here instead? I think it would make more sense to have that example here. Also probably add a print of node indices after the removal so after `remove_node(2)` call print(graph.node_indexes())` to show that the list has the hole.

Reply:
Sure.

Updated the docstring of PyDAG on [`6eb08de`](https://github.com/Qiskit/retworkx/pull/359/commits/6eb08de6f39106b5e97ff72d5a45daab8a2be6c0)

Added the jupyter example to both PyGraph and PyDiGraph on [`2032410`](https://github.com/Qiskit/retworkx/pull/359/commits/20324107942ac9871b970e0ac0f859c96fec8e88)

Reply:
```suggestion
/// Node indices will be reused on additions after removal. For example:
```